### DISCUSSION TITLE
Gocial.

### DISCUSSION THREAD

**Discussion by oofy123** [oofy123/Gocial#1] [General]
Gocial thing

**Comment by oofy123** [comment]
Yes

**Reply by theycallhermax** [reply]
ok

**Reply by oofy123** [reply]
E

**Comment by Bloctans** [comment]
nb;lkvjhhbvjhojbkj

**Reply by oofy123** [reply]
Hi

**Reply by oofy123** [reply]
We will be using cloudlink and the server code is hosted on replit

**Reply by theycallhermax** [reply]
ok

**Reply by theycallhermax** [reply]
wait can i make client too pls?

**Reply by oofy123** [reply]
What is your Replit username

**Reply by oofy123** [reply]
> wait can i make client too pls?

YES PLEASE bloc is better at backend

**Reply by oofy123** [reply]
You can do frontend

**Reply by oofy123** [reply]
And client if you want

**Reply by oofy123** [reply]
Alright md

**Reply by Bloctans** [reply]
frontend and client = same thing

**Reply by oofy123** [reply]
Remember the repl

**Reply by oofy123** [reply]
> frontend and client = same thing

So you will work with me

**Reply by theycallhermax** [reply]
> What is your Replit username

mdwalters

**Reply by Bloctans** [reply]
why are we using replit

**Reply by oofy123** [reply]
Oops wait confuse

**Reply by oofy123** [reply]
Idk

**Reply by oofy123** [reply]
We don’t need too

**Reply by oofy123** [reply]
To*

**Reply by Bloctans** [reply]
why dont we host an actual server

**Reply by theycallhermax** [reply]
we could use github codespaces /j

**Reply by oofy123** [reply]
Ok ok

**Reply by oofy123** [reply]
Yes to both

**Reply by oofy123** [reply]
Hosting server and codespaces

**Reply by theycallhermax** [reply]
> Ok ok

i was just joking, codespaces would automatically turn off, and we don't want someone to be writing something very important, then all the sudden, the server turns off and disconnects them

**Reply by theycallhermax** [reply]
@oofy123 should i make a github projects for todo list?

**Reply by theycallhermax** [reply]
i like keeping todo lists with gh projects

**Reply by theycallhermax** [reply]
WE NEED A LOGO

**Reply by theycallhermax** [reply]
@oofy123 logo

**Reply by oofy123** [reply]
Sorry I had to go to bed and phone died

**Reply by oofy123** [reply]
Someone also stole charger

**Reply by theycallhermax** [reply]
rip

**Reply by theycallhermax** [reply]
should we put this in a github org?

**Reply by theycallhermax** [reply]
@oofy123 gocial logo?

![image](https://user-images.githubusercontent.com/67456566/214609718-2b714ad5-79a3-48ac-8356-090577be51bd.png)

**Reply by oofy123** [reply]
> @oofy123 gocial logo?
> 
> 
> 
> ![image](https://user-images.githubusercontent.com/67456566/214609718-2b714ad5-79a3-48ac-8356-090577be51bd.png)
> 
> 

YES!!!!

**Reply by oofy123** [reply]
> should we put this in a github org?

There is one want an inv?

**Reply by theycallhermax** [reply]
> > @oofy123 gocial logo?
> > ![image](https://user-images.githubusercontent.com/67456566/214609718-2b714ad5-79a3-48ac-8356-090577be51bd.png)
> 
> YES!!!!

yay

**Comment by oofy123** [comment]
Discuss server client and other coding stuff by replying to this

**Reply by theycallhermax** [reply]
ok

**Reply by oofy123** [reply]
Dedicated reply thingy

**Reply by Bloctans** [reply]
uh hm ngrok + cf?

**Reply by theycallhermax** [reply]
hmm

**Reply by Bloctans** [reply]
my idea:
setup a cf page to fetch the current ngrok server (idk how i could fetch it)

**Reply by theycallhermax** [reply]
ok ig

**Reply by theycallhermax** [reply]
so... how about client?

**Reply by Bloctans** [reply]
just cf pages or gh pages

**Reply by theycallhermax** [reply]
no how do

**Reply by theycallhermax** [reply]
should i do with plain html or something?

**Reply by Bloctans** [reply]
S V E L T E

**Reply by theycallhermax** [reply]
ok

**Reply by theycallhermax** [reply]
so i'll just go create website now

**Reply by Bloctans** [reply]
i wont really be working on gocial but ok

**Reply by oofy123** [reply]
> i wont really be working on gocial but ok

Why not?

**Reply by oofy123** [reply]
Wait so no cloudlink??????

**Reply by oofy123** [reply]
Lol I’m ok with that but css isn’t my thing

**Reply by oofy123** [reply]
Languages I code in: python, html, Lua

**Reply by theycallhermax** [reply]
right after i get mdwalters-laptop back

**Reply by oofy123** [reply]
> right after i get mdwalters-laptop back

Did you create website

**Reply by theycallhermax** [reply]
no

**Reply by theycallhermax** [reply]
si

![image](https://user-images.githubusercontent.com/67456566/202312683-cb29fed4-4e2f-467b-8c83-a7cd0a2a0071.png)

**Reply by theycallhermax** [reply]
@oofy123 i was bored so i coded a simple server for gocial

**Reply by theycallhermax** [reply]
i hope you don't mind if it's written in node.js

**Reply by oofy123** [reply]
> i hope you don't mind if it's written in node.js

Nah I dont

**Reply by oofy123** [reply]
> @oofy123 i was bored so i coded a simple server for gocial

Thanks!

**Reply by oofy123** [reply]
> i hope you don't mind if it's written in node.js

We need to rewrite server backend in python md. Sorry.

**Reply by theycallhermax** [reply]
> > i hope you don't mind if it's written in node.js
> 
> We need to rewrite server backend in python md. Sorry.

i wasted my time 😐

**Reply by theycallhermax** [reply]
@oofy123 i created saved gocial posts

![image](https://user-images.githubusercontent.com/67456566/202678904-ebfbd18b-1f9f-42ba-a44f-7faebb89f916.png)

![image](https://user-images.githubusercontent.com/67456566/202679092-9ef96e22-f162-4e61-b0c7-74859bab5ea9.png)

**Reply by theycallhermax** [reply]
if saved posts existed for a longer time, it would have spam or me talking to myself

**Reply by oofy123** [reply]
> if saved posts existed for a longer time, it would have spam or me talking to myself

This is cool!

**Reply by oofy123** [reply]
> > > i hope you don't mind if it's written in node.js
> > 
> > 
> > We need to rewrite server backend in python md. Sorry.
> 
> i wasted my time 😐

Keep node server for now just for testing

**Comment by theycallhermax** [comment]
i didn't get mdwalters-laptop yet



and the people i returned it to lost it

**Reply by oofy123** [reply]
Wdym

**Reply by oofy123** [reply]
E

**Reply by theycallhermax** [reply]
the people (my school) gave me mdwalters-laptop, and i decided to put ubuntu on it, then i had to return it, and when i asked for it back the people said they didn't know where it was

**Reply by theycallhermax** [reply]
also sorry for making a new comment email is pain

**Reply by oofy123** [reply]
> also sorry for making a new comment email is pain

A

**Comment by theycallhermax** [comment]
dead

**Reply by oofy123** [reply]
> dead

No. I may or may not have taken a "small" break.

**Comment by theycallhermax** [comment]
i'm making a svelte client rn

**Reply by oofy123** [reply]
yay

**Comment by oofy123** [comment]
GUI preview exists now ![image](https://user-images.githubusercontent.com/77684463/224338188-c26152b8-1ca3-495f-8323-1952f57e2b15.jpeg)![image](https://user-images.githubusercontent.com/77684463/224338201-35045a32-c66a-41c5-9571-aeec9c1aa5e2.jpeg)

**Reply by oofy123** [reply]
yes

**Reply by oofy123** [reply]
Things we probably need to get working from this preview: posts, replies, pic upload

**Reply by oofy123** [reply]
Moderators need to review photos before you can post them

**Reply by theycallhermax** [reply]
where link

**Comment by oofy123** [comment]
hi im alive and i have a pc

**Reply by theycallhermax** [reply]
yay
